civihr_default_theme/scss/base/_variables.scss

```diff
@@ -10,38 +14,29 @@ $crm-header-height: 60px !default;
 
 // Grays
 // -----------------------------------------------------------------------------
-$gray-dark:           #555862 !default; // menu bg
-$gray-darker:         #4c4c5d !default; // text darker
-$gray:                #757b8e !default; // main text/lighter, menu item bg hover, footer logo
-$gray-light:          #ecedee !default; // body bg
-$gray-lighter:        #fbfbfc !default; // table odd row
-
-$light:               #fff !default; // well bg, text on dark bg hover
-$light-pale:          #c3cbd6 !default; // text on dark bg
-$light-gray:          #eeeded !default;
+$gray-darker:         #3e3e54 !default;
+$gray-dark:           #464354 !default;
+$gray:                #4d4d69 !default;
+$gray-light:          #9494a5 !default;
+$gray-lighter:        #c2cfd8 !default;
 
-$text-muted:          rgba($gray, 0.6);
+$light:               #fff !default;
+$light-pale:          #e8eef0 !default;
+$light-gray:          #f3f6f7 !default;
```

Comment:
also please check that the format that bootstrap follows is `$color-shade`, so it should be `$gray-light`. The problem is that `$gray-light` already exist, so you need to understand what kind of shade of gray we're dealing with here, so you can properly name it